src/MoBi.Presentation/Presenter/PathAndValueBuildingBlockPresenter.cs

```diff
@@ -1,303 +1,99 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
+﻿using System.Collections.Generic;
```

Comment:
There was a file level renaming done with these presenters. One was called PathWithValue... which doesn't match anything so this looks like a lot of changes, but really is not.